docs/ipsec-tunnel.md

```diff
@@ -0,0 +1,52 @@
+# IPSec Encryption of Tunnel Traffic with Antrea
+
+Antrea supports encrypting tunnel traffic across Nodes with IPSec ESP. At this
+moment, IPSec encyption works only for GRE tunnel (but not VXLAN, Geneve, and
+STT tunnel types).
+
+## Prerequisites
+
+IPSec requires a set of Linux kernel modules. Check the required kernel modules
+listed in the [strongSwan documentation](https://wiki.strongswan.org/projects/strongswan/wiki/KernelModules).
+Make sure the required kernel modules are loaded on the Kubernetes Nodes before
+deploying Antrea with IPSec encyption enabled.
+
+## Installation
+
+You can simply applying the [Antrea IPSec deployment yaml](/build/yamls/antrea-ipsec.yml)
+to deploy Antrea with IPSec encyption enabled. To deploy a released version of
+Antrea, pick a version from the [list of releases](https://github.com/vmware-tanzu/antrea/releases).
+For any given release `<TAG>` (e.g. `v0.1.0`), you can get the Antrea IPSec
```

Comment:
I don't think IPSec was available in 0.1.0...

Reply:
Changed to 0.3.0.